.travis.yml

```diff
@@ -49,3 +59,25 @@ script:
   - gimme -f 1.11.4
   - source ~/.gimme/envs/go1.11.4.env
   - go test ./...
```

Comment:
Should we remove the lines 49-61?

Reply:
Did you mean 59 - 61? I think we should keep it, as long as want go build / go test to continue to work. Also, it's still what we use to release/build images.

Reply:
Can you add a note what we're using bazel for then? I thought we were going to switch everything to bazel. I'd love to keep using go build/test wherever we can because they're much faster, but I also think it's good to be consistent with the build system.

Reply:
This prevents the problem of #638 where the bazel build is broken at HEAD. Bazel is needed especially when the proto definitions change.